components/ViewArticlePage/ViewsPanel.tsx

```diff
@@ -0,0 +1,34 @@
+import { Paper, Typography } from '@mui/material'
+
+const ViewsPanel = ({ viewCount }: { viewCount: number }) => {
```

Comment:
this component is the same as RevisionsPanel. so we can just abstract it out.
```
import { Paper, Typography } from '@mui/material'

enum CountType {
  REVISION = 'Revision',
  VIEWS = 'Views'
}

const ArticleCountPanel = ({ count, countType }: { count: number, countType: CountType }) => {
  return (
    <Paper
      elevation={2}
      sx={{
        height: 165,
        borderRadius: 3,
        marginBottom: '20px',
        padding: '12px 24px',
        display: 'flex',
        flexDirection: 'column',
        justifyContent: 'space-evenly',
      }}
    >
      <Typography fontSize={'21px'} fontWeight={700}>
        `Number of ${countType}:`
      </Typography>
      <Typography textAlign={'center'} fontSize={'28px'} fontWeight={400}>
        {count}
      </Typography>
      <Typography
        textAlign={'right'}
        color="#009DD7"
        sx={{ textDecoration: 'underline' }}
      >
        `View ${countType}&apos;s History`
      </Typography>
    </Paper>
  )
}

export default ArticleCountPanel
```

then when using it:
```
<ArticleCountPanel count={13} countType={CountType.REVISION} />
<ArticleCountPanel count={100} countType={CountType.VIEWS} />
```